workflows/URLScan_with_urlscanio_from_Microsoft_Teams/workflow.spec.yaml

```
@@ -0,0 +1,31 @@
+extension: workflow
+products: ["insightconnect"]
+name: URLScan_with_urlscanio_from_Microsoft_Teams
```

Update directory name workflow file name too

```suggestion
name: Enrich_URL_with_urlscan.io_from_Microsoft_Teams
```